download/src/lib.rs

```diff
@@ -132,6 +132,8 @@ pub fn download_to_path_with_backend(
     })
 }
 
+static USER_AGENT: &str = concat!("rustup/", env!("CARGO_PKG_VERSION"));
```

Comment:
Or we change the rules so that the download sub-crate gets the same version during the release process.  That'd just need an update to the release process described in `CONTRIBUTING.md`

Reply:
Or we merge the crates. Or we pass the value in from the location of use.